_data/people/Joseph87

```diff
@@ -0,0 +1,7 @@
+Username: Joseph87
+
+Name: Joseph
+
+Organization: UNAM School of Computing
+
+Why: Just chilling and learning GitHub and Python
```

Comment:
You'll want to make sure the keys here are lowercase, like:

`username: Joseph87`
`name: Joseph`

And so on. Let me know if you need help fixing these.